### PR TITLE
Minor Bugfixes

### PR DESCRIPTION
**Pull request by whmii**
This addresses a series of issues associated visual appearance across platforms and other.

### CODE REVIEW

File: source/assets/stylesheets/components/_docs-nav.scss [hunk 1]
```diff
@@ -11,17 +11,23 @@
...
-      overflow-y: scroll;
-      overflow-x: visible;
+      overflow: hidden;
 
       &::-webkit-scrollbar {
         width: 0 !important;
       }
+
+      .docs-nav__inner {
+        overflow-y: scroll;
```
Comment (tysongach) [inline source/assets/stylesheets/components/_docs-nav.scss]: Should this be `auto`? I believe using `scroll` will _always_ show scroll bars, whereas with `auto` they will only show if needed.

File: source/assets/stylesheets/components/_docs-nav.scss [hunk 2]
```diff
@@ -11,17 +11,23 @@
...
+      }
     }
   }
 }
 
+.docs-nav__inner {
+  width: calc(100% + 18px);
```
Comment (tysongach) [inline source/assets/stylesheets/components/_docs-nav.scss]: Where does `18px` come from?

File: source/partials/_doc_item_example.slim [hunk 3]
```diff
@@ -2,20 +2,14 @@ h5 Example
...
 - item.example.each do |example|
 
-  - if example.description.present?
-    p
-      = example.description
-
   - if example.type == "scss"
     figure
       figcaption SCSS
-      pre
```
Comment (tysongach) [inline source/partials/_doc_item_example.slim]: Why remove the `pre` here? Aren’t these code examples preformatted text?

File: source/assets/stylesheets/components/_docs-nav.scss (hunk 2 above)
Comment (whmii) [inline source/assets/stylesheets/components/_docs-nav.scss]: This is to accommodate the rando width of PC firefox's scroll bars

File: source/partials/_doc_item_example.slim (hunk 3 above)
Comment (whmii) [inline source/partials/_doc_item_example.slim]: The  `pre` is already being output by the ` code()` method. It was resulting in multiple `pre`s.

here's a copy of some code from the live site.

```html
<figure>
  <figcaption>SCSS</figcaption>
  <pre>
    <pre class="syntax-highlight scss">
      <code>
        $neat-grid:(
          columns: 12,
          gutter: 20px,
        );
      </code>
    </pre>
  </pre>
</figure>
```

**Pull request merged**